conf/main/grakn.properties

```diff
@@ -140,16 +242,5 @@ spark.master=local[*]
 spark.serializer=org.apache.spark.serializer.KryoSerializer
 spark.ui.showConsoleProgress=false
 
-####################################
-# Apache Cassandra InputFormat configuration
-####################################
-cassandra.input.partitioner.class=org.apache.cassandra.dht.Murmur3Partitioner
-cassandra.input.predicate=0c00020b0001000000000b000200000000020003000800047fffffff0000
-cassandra.input.columnfamily=edgestore
-cassandra.range.batch.size=4096
-cassandra.thrift.framed.size_mb=1024
-
-####################################
-# Hadoop Cluster configuration     #
-####################################
+############################# Hadoop Cluster configuration #############################
 #fs.defaultFS=hdfs://lxd-cluster2-cassandra1:9000
```

Comment:
What should it be- `hadoop://123.123.123.123:9000`? Are the `123`s just placeholders?